src/parkapi_sources/converters/kienzler/models.py

```diff
@@ -42,4 +45,5 @@ def to_realtime_parking_site(self) -> RealtimeParkingSiteInput:
             realtime_data_updated_at=datetime.now(tz=timezone.utc),
             realtime_capacity=self.sum_boxes,
             realtime_free_capacity=self.bookable,
+            realtime_opening_status=UnsetValue,
```

Comment:
This is the default value and therefore not necessary.

Reply:
Removed